### PR TITLE
Linux Support

### PR DESCRIPTION
**Pull request by Mubashir-Ansari**

### CODE REVIEW

**Comment by Mubashir-Ansari** [comment]
@wajeehmisbahkhan Request for review.

File: Docs/Snapcraft.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Linux Electron App build, Snapcraft Release, and Actions
```
Comment (wajeehmisbahkhan) [inline Docs/Snapcraft.md]: Rename file to LINUX.md

File: Docs/Snapcraft.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+### Actions
+
+This is a Github Action that can be used to publish [snap
+packages](https://snapcraft.io) to the Snap Store built by [snapcore](https://github.com/snapcore/action-publish).
+
+![carbon(9)](https://user-images.githubusercontent.com/42158443/147774289-49e4197d-ddd8-4e00-9a94-e1fbd55a820b.png)
+
+This action is already written in [.github/workflows/sanpcraft.yaml](https://github.com/Efshal/boilerplate-monorepo/blob/main/.github/workflows/snapcraft-publish.yml), you have to first produce data using command below:
```
Comment (wajeehmisbahkhan) [inline Docs/Snapcraft.md]: Update to correct links

File: main.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
+// "use strict";
```
Comment (wajeehmisbahkhan) [inline main.js]: Remove dead code and update file to TypeScript instead of JS

**Review by wajeehmisbahkhan — CHANGES_REQUESTED** [review]

File: .github/workflows/snapcraft-publish.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name: Snapcraft Publish Action
```
Comment (wajeehmisbahkhan) [inline .github/workflows/snapcraft-publish.yml]: Rename this file to `deploy-linux` and create another action which only builds the linux package but does not upload it. This should run on pull request.

File: Docs/LINUX.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+### Actions
+
+This is a Github Action that can be used to publish [snap
+packages](https://snapcraft.io) to the Snap Store built by [snapcore](https://github.com/snapcore/action-publish).
+
+![](images/../../images/action.png)
+
+This action is already written in [.github/workflows/sanpcraft.yaml](https://github.com/Efshal/boilerplate-monorepo/blob/main/.github/workflows/snapcraft-publish.yml), you have to first produce data using command below:
```
Comment (wajeehmisbahkhan) [inline Docs/LINUX.md]: update URL

**Review by wajeehmisbahkhan — CHANGES_REQUESTED** [review]

**Comment by Mubashir-Ansari** [comment]
@wajeehmisbahkhan changes done.

**Review by wajeehmisbahkhan — APPROVED** [review]

**Pull request merged**